packages/next-swc/crates/core/src/server_actions.rs

```diff
@@ -277,7 +294,7 @@ impl<C: Comments> VisitMut for ServerActions<C> {
     fn visit_mut_stmt(&mut self, n: &mut Stmt) {
         n.visit_mut_children_with(self);
 
-        if self.in_module {
+        if self.in_module || self.in_prepass {
```

Comment:
This will prevent finding nested actions.

Reply:
But it seems like it's intended?

Reply:
Yes this one is intended, in the prepass we can skip some unnecessary steps to improve performance.